vms/platformvm/state/camino.go

```diff
@@ -243,8 +259,9 @@ func newCaminoState(baseDB *versiondb.Database, validatorsDB *prefixdb.Database,
 		depositOffersList: linkeddb.NewDefault(depositOffersDB),
 
 		// Deposits
-		depositsCache: depositsCache,
-		depositsDB:    prefixdb.New(depositsPrefix, baseDB),
+		depositsCache:         depositsCache,
+		depositsDB:            depositsDB,
```

Comment:
Why one is extracted into variable but not another. I'd prefer to have all of them inline if there is no further references and keep consistent...

I don't insist you to change it but it might help ;)